core/blockchain_insert.go

```diff
@@ -44,7 +44,11 @@ func (st *insertStats) report(chain []*types.Block, index int, snapDiffItems, sn
 	var (
 		now     = mclock.Now()
 		elapsed = now.Sub(st.startTime)
+		mgasps  = float64(st.usedGas) * 1000 / float64(elapsed)
```

Comment:
We should try to avoid the zero division.

e.g.,
`		elapsed = now.Sub(st.startTime) + 1 // plus 1 ns to prevent zero division
`